packages/devtools-reps/configs/development.json

```diff
@@ -0,0 +1,23 @@
+{
+  "title": "Reps",
+  "hotReloading": true,
+  "defaultURL": "https://nchevobbe.github.io/demo/console-test-app.html",
+  "environment": "development",
+  "logging": {
+    "client": false,
+    "firefoxProxy": false,
+    "actions": false
+  },
+  "theme": "light",
+  "firefox": {
+    "webSocketConnection": false,
+    "host": "localhost",
+    "webSocketPort": 9000,
+    "tcpPort": 6080,
+    "mcPath": "./firefox"
+  },
+  "development": {
+    "serverPort": 8000
+  },
+  "features": {}
+}
```

Comment:
can we get rid of this?

Reply:
we need it for local reps development right (if we want different settings than the one for the debugger) ?